rabit/rabit/Goal/GoalAdd/GoalAddViewController.swift

```diff
@@ -78,6 +78,10 @@ final class GoalAddViewController: UIViewController {
         bind()
     }
     
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        view.endEditing(true)
+    }
```

Comment:
오 이런 메소드가 원래 UIViewController에 있었군요? 처음 알았습니다ㅋㅋㅋ
그런데 override method라, 수퍼클래스 메소드도 한 번 호출해주는게 적절할 것 같습니다!
```suggestion
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        super.touchesBegan(touches, with: event)
        view.endEditing(true)
    }
```

Reply:
오우 깜빡했어요 ㅋㅋㅋ 감사합니다